app/jobs/bulk_sync_courses_to_find_job.rb

```diff
@@ -5,6 +5,9 @@ class SearchAndCompareRequestError < StandardError
   end
 
   def perform
+    RequestStore.store[:job_id] = provider_job_id
```

Comment:
I get that RequestStore is storing the `job_id` and `queue_name`. I'm guessing that the `customize_event` you defined in `production.rb` passes on these values. Is the event only sent if values exist?

Reply:
No, if they don't exist they'll just be nil.

Reply:
Kibana has a way to filter events based on the non-existence of a field. Sounds like it might be better to _not_ include this if there is no data for it (in `production.rb`).